commander/src/utils/download.ts

```diff
@@ -14,9 +14,34 @@
  *
  */
 import * as axios from 'axios';
+import crypto from 'crypto';
 import fs from 'fs-extra';
+import * as tar from 'tar';
 import { dateDiff, getDownloadedFileInfo } from './core/commons';
-import { exec, ExecResult } from './worker-process';
+
+export const verifyChecksum = async (
+	filePath: string,
+	expectedChecksum: string,
+): Promise<boolean> => {
+	const shasum = crypto.createHash('sha256');
```

Comment:
prefer to use `hash` and `bufferToHex` from `lisk-cryptography`
We shouldn't try not to use `crypto` directly

Reply:
the problem with using the `hash` function was that its not a buffer.. its stream here.. so

Reply:
we can read the file and then create the hash. we don't need to do it while streaming i think